src/app/components/logo/SmallLogo.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react'
+import { FC } from 'react'
+import logo from './images/logo.png'
```

Comment:
This file is already in a repo logo512.png no need to duplicate it. SVG would be better here in general.

Reply:
Right. Deduplicated / simplified.